stroeer/core/v1/core_article_service.proto

```diff
@@ -58,12 +61,36 @@ service ArticleService {
 * ```
 * | Field name       | Type                | Description                                                 |
 * |------------------|---------------------|-------------------------------------------------------------|
-* | `id`             | `id`                | [required] Unique ID of the article defined by the content management system |
+* | `id`             | `int64`             | [required] Unique id of the article to be fetched.          |
 */
 message GetArticleRequest {
   int64 id = 1;
 }
 
+/**
+* # `⚙︎ BatchGetArticle`
+* ```protobuf
+* rpc BatchGetArticles (BatchArticlesRequest) returns (BatchArticlesResponse) {}
+* ```
+*
+* returns multiple [`stroeer.core.v1.Article`](Article.html) for the given `ids`. The
+* ordering of items will the same ordering as the `ids` requested.
+* If an `id` does not exists, it is omitted in the result (no error will be raised).
+*
+* There is a maximum of 100 items that can be queried in one batch.
```

Comment:
How do we deal with the 16MB limit (implementation detail)?

Reply:
this is `163 kB` per item ... should be plenty.

... buuuut, If we really get that far, there is little we can do. I can think of one on these:

- reduce the max item count (to 75 or even less to have some margin)
- expose DDB's `UnprocessedKeys` to the consumer and let him handle this issue (also relevant if table was throttled)